catalog/locales/en/out.gotext.json

```diff
@@ -0,0 +1,30 @@
+{
+    "language": "en",
+    "messages": [
+        {
+            "id": "not implemented",
+            "message": "not implemented",
+            "translation": "This method is not implemented yet."
+        },
+        {
+            "id": "{MethodName} is not implemented",
+            "message": "{MethodName} is not implemented",
+            "translation": "{MethodName} is not implemented",
+            "placeholders": [
+                {
+                    "id": "MethodName",
+                    "string": "%[1]s",
+                    "type": "string",
+                    "underlyingType": "string",
+                    "argNum": 1,
+                    "expr": "methodName"
+                }
+            ]
+        },
+        {
+            "id": "This method is not implemented yet.",
+            "message": "This method is not implemented yet.",
+            "translation": ""
+        }
+    ]
+}
```

Comment:
Can we add \n to those two files?

Reply:
I do not see positivities to do this in go text
https://github.com/golang/text/blob/23ae387dee1f90d29a23c0e87ee0b46038fbed0e/message/pipeline/pipeline.go#L362

should we format it using external tools like `jq` ?

Reply:
Well, I would use `cat`, `echo` or something :)